src/text-editor.coffee

```diff
@@ -2848,6 +2858,7 @@ class TextEditor extends Model
 
       didInsertEvent = {text, range}
       @emitter.emit 'did-insert-text', didInsertEvent
+      @emitter.emit 'did-paste-text', didInsertEvent
```

Comment:
What does this new feature exactly add, since all it seems to do is emit the same message as `'did-insert-text'`?